gdnative-core/src/core_types/color.rs

```diff
@@ -47,6 +48,68 @@ impl Color {
             a: self.a + (weight * (other.a - self.a)),
         }
     }
+    #[inline]
+    pub fn blend(&self, other: &Color) -> Color {
+        Color::from_sys(unsafe { (get_api().godot_color_blend)(self.sys(), other.sys()) })
+    }
+
+    #[inline]
+    pub fn contrasted(&self) -> Color {
+        Color::from_sys(unsafe { (get_api().godot_color_contrasted)(self.sys()) })
+    }
+    #[inline]
+    pub fn darkened(&self, amount: f32) -> Color {
+        Color::from_sys(unsafe { (get_api().godot_color_darkened)(self.sys(), amount) })
+    }
+    #[inline]
+    pub fn from_hsv(h: f32, s: f32, v: f32, a: f32) -> Color {
+        let color = Color::rgba(0.0, 0.0, 0.0, 0.0);
+        Color::from_sys(unsafe { (get_api().godot_color_from_hsv)(color.sys(), h, s, v, a) })
+    }
+    #[inline]
+    pub fn gray(&self) -> f32 {
+        // Implemented as described in godot docs
+        (self.r + self.b + self.g) / 3.0
+    }
+
+    #[inline]
+    pub fn inverted(&self) -> Color {
+        // Implementation as described in godot docs.
+        Color {
+            r: 1.0f32 - self.r,
+            g: 1.0f32 - self.g,
+            b: 1.0f32 - self.b,
+            a: self.a,
+        }
+    }
+
+    pub fn to_abgr32(&self) -> i32 {
+        unsafe { (get_api().godot_color_to_abgr32)(self.sys()) }
+    }
+
+    pub fn to_abgr64(&self) -> i32 {
+        unsafe { (get_api().godot_color_to_abgr64)(self.sys()) }
+    }
+
+    pub fn to_argb32(&self) -> i32 {
+        unsafe { (get_api().godot_color_to_argb32)(self.sys()) }
+    }
+
+    pub fn to_argb64(&self) -> i32 {
```

Comment:
This type must be `u64`.

From the official doc:

> **int to_argb64 ( )**
> 
> Returns the color converted to a 64-bit integer in ARGB format (each word represents a color channel). ARGB is more compatible with DirectX.

Reply:
Good catch, I'll make the necessary changes once the u64 or [u16; 4] is settled.

Reply:
We settled for unsigned integers, so this would be `u64`.